frame/pablo/src/lib.rs

```diff
@@ -324,10 +360,19 @@ pub mod pallet {
 					Self::deposit_event(Event::PoolCreated { pool_id, owner: pool.owner.clone() });
 					Ok(pool_id)
 				},
+				PoolConfiguration::ConstantProduct(constant_product_pool_info) =>
+					Uniswap::<T>::do_create_pool(
```

Comment:
Uniswap module can be called in the relevant places including the Pallet AMM impl functions